src/components/NavBar/UkBanner.tsx

```diff
@@ -0,0 +1,78 @@
+import { Trans } from '@lingui/macro'
+import { useOpenModal } from 'state/application/hooks'
+import { ApplicationModal } from 'state/application/reducer'
+import styled from 'styled-components'
+import { ButtonText, ThemedText } from 'theme/components'
+import { Z_INDEX } from 'theme/zIndex'
+
+export const UK_BANNER_HEIGHT = 64
+export const UK_BANNER_HEIGHT_MD = 112
+export const UK_BANNER_HEIGHT_SM = 136
+
+const BannerWrapper = styled.div`
+  position: relative;
+  display: flex;
+  background-color: ${({ theme }) => theme.surface1};
+  padding: 20px;
```

Comment:
can we reduce padding on mobile? Takes up a lot of the screen rn
![Screenshot 2023-10-05 at 2 43 55 PM](https://github.com/Uniswap/interface/assets/11512321/e20d4687-2c6d-4660-aea8-ab315b1d8de9)

Reply:
for some reason this is the way design has it in the figma specifically for mobile too